### ISSUE TITLE
[Feature] Use .brainglobe directory to store deformations

### ISSUE THREAD

**Issue by PolarBean** [Neural-Systems-at-UIO/CCF_translator#11]
would .brainglobe/deformations be a good directory from me to store the data @adamltyson?

**Comment by PolarBean** [comment]
Alternatively I could store the deformation in the directory of the atlas, in the same folders as the atlas files. I'll review the v2 spec and start conforming to this

**Comment by adamltyson** [comment]
Yeah, for now. We can/should revisit once the API itself is updated. 

> add just a _v1.0 string 

I assume you mean add the actual version, not just `_v1.0`!

**Comment by PolarBean** [comment]
well all the versions now will be v1.0! as it would just be versioning for the deformation matrix. But yeah in the future if the deformations are updated we would update the version number.

One issue is that there are versions of spaces and versions of deformations... This might get confusing

**Comment by adamltyson** [comment]
Oh yeah, I misunderstood. Should a deformation version be a combination of space versions? I.e. a specific deformation version for specific versions of source and target spaces?

**Comment by PolarBean** [comment]
but then again I assume the space won't update without changing it's name entirely??

**Comment by PolarBean** [comment]
if there was theoretically a ccfv4 based on MRI. would this still be allen_mouse in the brainglobe api
